File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_num_psets.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_session_get_num_psets(var_0, var_1, var_2, var_3)
       call pmpi_session_get_num_psets(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_num_psets.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Session var_0;
    MPI_Info var_1;
    int *var_2;
    int ret;
    /* calls */
    ret = MPI_Session_get_num_psets(var_0, var_1, var_2);
    ret = PMPI_Session_get_num_psets(var_0, var_1, var_2);
    return 0;
}
